src/libtego/source/core/ContactUser.h

```diff
@@ -75,14 +64,12 @@ class ContactUser : public QObject
         Offline,
         RequestPending,
         RequestRejected,
-        Outdated
+        Outdated    // this doesn't seem to actually be used
```

Comment:
then is there any reason in keeping it? i can't find anything that references it by value (4) either

Reply:
I tried to keep as much of the original code in-tact during this refactor so didn't delete *everything* I didn't think was used. Another pass on this could happen to purge dead code (like any remaining Qml annotations on code living in libtego for instance).